models/Translation/Dao.php

```diff
@@ -173,8 +185,28 @@ public function createOrUpdateTable()
                           `text` text DEFAULT NULL,
                           `creationDate` int(11) unsigned DEFAULT NULL,
                           `modificationDate` int(11) unsigned DEFAULT NULL,
+                          `userOwner` int(11) unsigned DEFAULT NULL,
+                          `userModification` int(11) unsigned DEFAULT NULL,
                           PRIMARY KEY (`key`,`language`),
                           KEY `language` (`language`)
                         ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;");
     }
+
+    protected function updateModificationInfos()
```

Comment:
```suggestion
    protected function updateModificationInfos(): void
```

Reply:
we don't add void return type-hints for simplicity.